src/data-pipeline/etl-common/build.gradle

```diff
@@ -24,8 +24,8 @@ apply from: "${rootProject.projectDir}/gradle/config/scripts/coverage.gradle"
 group = "$group"
 version = "$projectVersion"
 
-sourceCompatibility = JavaVersion.VERSION_17
-targetCompatibility = JavaVersion.VERSION_17
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
```

Comment:
should you update github actions for using Java 11 too?

Reply:
we can keep using java17 in github actions.  Java17 can compile java11 code.